xml/operations-monitoring-crt_change_compute_host.xml

```diff
@@ -56,7 +44,7 @@
   </listitem>
```

Comment:
I cannot comment on lines 33 and 27 directly, so it goes here ;-):

The long lines starting with &lt;screen>{"aggregation_params_map do not break correctly in the PDF. Two suggestions:

* replace "," with ", " (comma+space) to allow line breaks after the comma (AFAIK this is still valid json)
* ommit large parts of the string by replacing it with [...]